TA_Scheduler/views/account/create.py

```diff
@@ -1,7 +1,31 @@
 from django.shortcuts import render
 from django.views import View
+from TA_Scheduler.models import Account
+from TA_Scheduler.utilities.AccountUtil import AccountUtil
 
 
 class CreateAccount(View):
     def get(self, request):
-        return render(request, "account/create.html")
+        return render(request, "account/create.html", {"message": " "})
+
+    def post(self, request):
+        preexisting = True
+        username = request.POST['username']
+        if AccountUtil.getAccountByUsername(username) is None:
+            preexisting = False
+        if preexisting:
+            return render(request, "account/create.html", {"message": "username '" + username + "' is already in use"})
+        else:
```

Comment:
`preexisting` is not necessary here. you can just combine the two if statements

Reply:
okay, i'll fix it